### PR TITLE
[misc] Gene table view minor cosmetic improvement.

### PR DESCRIPTION
**Pull request by veronikaslc**

### CODE REVIEW

**Comment by veronikaslc** [comment]
This is a cosmetic change to improve the view look of genes table. @AlexLapena @a-gvo please review

File: components/widgets/resources/src/main/resources/templates/geneVariantMacros.vm [hunk 1]
```diff
@@ -151,7 +151,7 @@ $doc.display($prop.getName(), 'view', $fakeObj )##
 #set($variantObjects = $doc.getObjects($variantClass.name, "genesymbol", $geneObj.getProperty('gene').value))
 |(% class="row-count" %)$geneNumber##
 #foreach($prop in $geneObj.xWikiClass.properties)##
-|(% class="$prop.translatedPrettyName gene gene-$geneObjNumber #if ($prop.name == 'gene')mandatory#end"##
+ |(% class="$prop.translatedPrettyName gene gene-$geneObjNumber #if ($prop.name == 'gene')mandatory#end"##
```
Comment (sdumitriu) [inline components/widgets/resources/src/main/resources/templates/geneVariantMacros.vm]: Are you sure this is needed?

File: components/widgets/resources/src/main/resources/resources/uicomponents/widgets/extraGeneVariantData.css [hunk 2]
```diff
@@ -79,6 +79,8 @@
 }
 .gene-table td.variant.moreinfo {
   position: relative;
+  height: 2.5em;
```
Comment (sdumitriu) [inline components/widgets/resources/src/main/resources/resources/uicomponents/widgets/extraGeneVariantData.css]: I'd use `min-height` instead.

File: components/widgets/resources/src/main/resources/templates/geneVariantMacros.vm (hunk 1 above)
Comment (sdumitriu) [inline components/widgets/resources/src/main/resources/templates/geneVariantMacros.vm]: Nbsps aren't the right way of fixing that. Why not add some padding?

Comment (veronikaslc) [inline components/widgets/resources/src/main/resources/templates/geneVariantMacros.vm]: indeed, will do

**Pull request merged**